incubator/couchdb/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: couchdb
-version: 1.1.2
+version: 1.1.3
```

please bump to 1.2.0
```suggestion
version: 1.2.0
```

---

Done :)